docs/recipes/svelte.md

```diff
@@ -0,0 +1,68 @@
+# Usage with Svelte
+
+XState integrates great with Svelte, and especially [Svelte stores!](https://svelte.dev/docs#svelte_store)
+
+The only gotcha is that you'll need this extra package.
+```js
+npm install @rollup/plugin-replace --save-dev
+```
+Then add this to the plugins array in rollup.config.js.
+```js
+replace({ 
+    'process.browser': true
```

Comment:
Is this needed though? 😉 XState doesn't contain such a thing in its source code

Reply:
i don't *really* understand what it's doing but it fixes this error and makes it work
environment.js:1 Uncaught ReferenceError: process is not defined

Reply:
It replaces `process.browser` in the source code with `true` - it's just a simple string to string replacement. Our `environment.js` file doesn't contain this string though - it only includes `process.env.NODE_ENV`:
https://unpkg.com/xstate@4.9.1/lib/environment.js

Reply:
If it doesn't work without it, we can keep it here for now and investigate later.

Reply:
I'm just worried that this is ill-advised and thus will be confusing for people. To the best of my knowledge - this is not required. The problem was about `process.env.NODE_ENV` not `process.browser`, but handling the former has been removed from the suggestion altogether. I suspect the whole initial suggestion (containing both `process.env.NODE_ENV` and `process.browser`) was copied from somewhere and hasn't been adjusted to the actual needs of XState.

If this really doesn't work cause of `process.browser` being in the source code I would appreciate seeing a repro case. The provided (@tivac) repro is failing because of `process.env.NODE_ENV` being in the bundle (which is supposed to be there as we want to differentiate dev & prod code paths):
<img width="1677" alt="Screenshot 2020-05-03 at 00 31 48" src="https://user-images.githubusercontent.com/9800850/80893788-c3419800-8cd5-11ea-91e3-205c90eee3bb.png">
So from my perspective, the part that has been removed (about the need to replace `process.env.NODE_ENV`) made much more sense to be included than what has been left as part of this PR.

Reply:
@Andarist thank you for pushing back on this, you are totally right.